src/App.tsx

```diff
@@ -1,61 +1,112 @@
+import { useState } from 'react';
+import { Todo } from './services/types';
+import { PreparedTodo } from './services/types';
+import { TodoList } from './components/TodoList';
+import { userSearcher } from './services/helpers';
+import { generateId } from './services/helpers';
+
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+const todos: Todo[] = todosFromServer;
+
+const preparedTodo: PreparedTodo[] = todos.map(todo => ({
+  id: todo.id,
+  title: todo.title,
+  completed: todo.completed,
+  user: userSearcher(todo.userId),
+}));
+
+export const App: React.FC = () => {
+  const [title, setTitle] = useState<string>('');
+  const [hasTitleError, setHasTitleError] = useState<boolean>(false);
+  const [userId, setUserId] = useState<number>(0);
+  const [hasUserIdError, setHasUserIdError] = useState<boolean>(false);
+  const [deals, setDeals] = useState(preparedTodo);
```

Comment:
and here you should add type
```suggestion
  const [deals, setDeals] = useState<PreparedTodo[]>(preparedTodo);
```